api/base/filters.py

```diff
@@ -215,6 +216,8 @@ def convert_value(self, value, field):
                 )
         elif isinstance(field, self.LIST_FIELDS):
             return value
+        elif isinstance(field, (RelationshipField, TargetField)):
```

Comment:
`self.LIST_FIELDS` is a tuple of one that doesn't appear to be used anywhere outside of this file (aside from, perhaps, somewhere inside of Django). Is there any reason not to put `RelationshipField, TargetField` into `LIST_FIELDS`, or to do the comparison together (as they all return `value`)?